src/services/web3/governance/governance.ts

```diff
@@ -82,5 +82,5 @@ export const getUnstakeTimer = async (user: string) => {
   );
   const locks = await govContract.voteLocks(user);
   const time = Number(locks) * 1000;
-  return time < 0 ? 0 : time;
+  if (time > 0) return time;
```

Comment:
please be explicit about what you return if time <= 0

Reply:
undefined is used in other places, you preffer -1?

Reply:
it's ok to return undefined, but please explicitly write `return undefined`